### PR TITLE
fixed viewport

### PR DESCRIPTION
**Pull request by hyphone**
On mobile clients everything was not scaled properly.

I added a viewport meta definition that fixes this but I'm not sure if this is the right way to do here.

### CODE REVIEW

File: app/index.html [hunk 1]
```diff
@@ -1,7 +1,14 @@
...
-  <title>Task Editor</title>
+  <head>
+    <base target="_blank">
+    <meta charset="UTF-8">
+    <title>Simple Task Editor</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" media="all" href="dist.css">
+  </head>
   <body>
+    <div class="sn-component" id="simple-task-editor"></div>
```
Comment (mobitar) [inline app/index.html]: @hyphone what's the basis for this line?

Comment (hyphone) [reply app/index.html]: @mobitar none really - I just played around and thought maybe this element with the sn-component class would be necessary to embed the component properly (and perhaps scaling would magically work). Should I remove it and make a new proper pull request with just the viewport?

Comment (mobitar) [reply app/index.html]: Yes you can remove it in the same PR..

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "sn-simple-task-editor",
-  "version": "1.3.5",
+  "version": "1.3.6",
```
Comment (mobitar) [inline package.json]: Could you revert this as well? Sorry should have mentioned this earlier. We handle versioning ourselves when we release it.

Comment (hyphone) [reply package.json]: sure, should have known that

Comment (mobitar) [reply package.json]: package-lock.json should be omitted completely from the PR as well.

**Comment by mobitar** [comment]
Thanks! This will be deployed in the near future.

**Pull request merged**